extensions/wikidata/module/scripts/wikibase-manager.js

```diff
@@ -188,3 +188,61 @@ WikibaseManager.fetchManifestFromURL = function (manifestURL, onSuccess, onError
   });
 };
 
+var wikibaseLogoURLCache = {
+  data: {},
+  exist: function (url) {
+    return wikibaseLogoURLCache.data.hasOwnProperty(url);
+  },
+  get: function (url) {
+    return wikibaseLogoURLCache.data[url];
+  },
+  set: function (url, cachedData) {
+    wikibaseLogoURLCache.data[url] = cachedData.responseJSON.query.general.logo;
+  }
+};
+
+
+// Retrives the wikibaseName instance site info, if wikibaseName is empty returns the selected wikibase site info
+WikibaseManager.retrieveLogoUrlFromSiteInfo = function(onSuccess, onError, wikibaseName) {
+  var params = {
+    action: 'query',
+    meta: 'siteinfo',
+    format: 'json'
+  };
+  let wikibase = (wikibaseName) ? WikibaseManager.wikibases[wikibaseName] : WikibaseManager.getSelectedWikibase();
+  const url = wikibase.mediawiki.api;
+  return $.ajax({
+    url: url,
+    data: params,
+    dataType: "jsonp",
+	cache: true,
+    timeout: 1000,
+    cache: true,
```

Comment:
LGTM warns us of a duplicate property `cache: true` here, it would make sense to remove it.